modules/study/src/main/PgnDump.scala

```diff
@@ -65,12 +77,15 @@ final class PgnDump(
       val opening = chapter.opening
       val genTags = List(
         Tag(_.Event, s"${study.name}: ${chapter.name}"),
-        Tag(_.Site, chapterUrl(study.id, chapter.id)),
         Tag(_.Variant, chapter.setup.variant.name.capitalize),
         Tag(_.ECO, opening.fold("?")(_.eco)),
         Tag(_.Opening, opening.fold("?")(_.name)),
         Tag(_.Result, "*"), // required for SCID to import
         annotatorTag(study)
+      ) ::: (!study.isRelay).so(
+        List(
+          Tag(_.Site, chapterUrl(study.id, chapter.id))
```

Comment:
Isn't `Site` a required tag? Is it guaranteed that it will always be set in the broadcast?

Reply:
Not all pgns provided contain the Site tag, and I don't see anything having problems due to the lack of the tag
I even went back to check and didn't see anything.

Reply:
I'm talking about the PGN standard. Though presumably most software will handle it fine. But there probably is some software that will refuse to load PGNs without it, so I think it's best that we ensure a Site tag is always set. From your screenshot, it looks like the way it previously worked already took the Site tag of the broadcast and only used the chapter URL as a fallback.

Reply:
ok return Tag Site
LCC source (for example) not have tag Site